packages/@aws-cdk/aws-codebuild/lib/linux-gpu-build-image.ts

```diff
@@ -0,0 +1,141 @@
+    '1.4.0-gpu-py36-cu101-ubuntu16.04');
+  /** PyTorch 1.5.0 GPU training image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_5_0_TRAINING = LinuxGpuBuildImage.awsDeepLearningContainersImage('pytorch-training',
+    '1.5.0-gpu-py36-cu101-ubuntu16.04');
+  /** PyTorch 1.5.0 GPU inference image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_5_0_INFERENCE = LinuxGpuBuildImage.awsDeepLearningContainersImage('pytorch-inference',
+    '1.5.0-gpu-py36-cu101-ubuntu16.04');
+
+  /** MXNet 1.4.1 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_MXNET_1_4_1 = LinuxGpuBuildImage.awsDeepLearningContainersImage('mxnet-training',
+    '1.4.1-gpu-py36-cu100-ubuntu16.04');
+  /** MXNet 1.6.0 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_MXNET_1_6_0 = LinuxGpuBuildImage.awsDeepLearningContainersImage('mxnet-training',
+    '1.6.0-gpu-py36-cu101-ubuntu16.04');
+
+  /**
+   * Returns a Linux GPU build image from AWS Deep Learning Containers.
+   *
+   * @param repositoryName the name of the repository,
+   *   for example "pytorch-inference"
+   * @param tag the tag of the image, for example "1.5.0-gpu-py36-cu101-ubuntu16.04"
+   * @param account the AWS account ID where the DLC repository for this region is hosted in.
+   *   In many cases, the CDK can infer that for you, but for some newer region our information
+   *   might be out of date; in that case, you can specify the region explicitly using this optional parameter
+   * @see https://aws.amazon.com/releasenotes/available-deep-learning-containers-images
+   */
+  public static awsDeepLearningContainersImage(repositoryName: string, tag: string, account?: string): IBuildImage {
+    return new LinuxGpuBuildImage(repositoryName, tag, account);
+  }
+
+  public readonly type = 'LINUX_GPU_CONTAINER';
+  public readonly defaultComputeType = ComputeType.LARGE;
+  public readonly imageId: string;
+  public readonly imagePullPrincipalType?: ImagePullPrincipalType = ImagePullPrincipalType.SERVICE_ROLE;
+  public readonly binder?: IBuildImageBinder;
+
+  private constructor(repositoryName: string, tag: string, account: string | undefined) {
+    const mappingName = 'AwsDeepLearningContainersRepositoriesAccounts';
+    const accountExpression = account ?? core.Fn.findInMap(mappingName, core.Aws.REGION, 'account');
+    this.imageId = `${accountExpression}.dkr.ecr.${core.Aws.REGION}.${core.Aws.URL_SUFFIX}/${repositoryName}:${tag}`;
+    this.binder = {
+      bind(scope: core.Construct, project: IProject): BuildImageConfig {
+        if (!account) {
+          const scopeStack = core.Stack.of(scope);
+          // Unfortunately, the account IDs of the DLC repositories are not the same in all regions.
+          // Because of that, use a (singleton) Mapping to find the correct account
+          if (!scopeStack.node.tryFindChild(mappingName)) {
```

Comment:
Feels like this whole block ought to be a utility somewhere.

At least getting the map out of `RegionInfo` could be, no?

Something like:

```
const mapping = RegionInfo.regionMap(FactName.DLC_REPOSITORY_ACCOUNT);
```
?

Reply:
Done.